clang/lib/Sema/SemaExceptionSpec.cpp

```diff
@@ -979,6 +979,11 @@ bool Sema::CheckOverridingFunctionExceptionSpec(const CXXMethodDecl *New,
   if (isa<CXXDestructorDecl>(New) && New->getParent()->isDependentType())
     return false;
 
+  // CWG1351: if either of the old function or the new function is defined as
+  // deleted, we don't need this check.
+  if (Old->isDeleted() || New->isDeleted())
```

Comment:
Either they both have to be deleted or not. We should diagnose otherwise. So I think checking both is redundant.